andy/src/test/java/integration/RunConfigurationTests.java

```diff
@@ -0,0 +1,120 @@
+package integration;
+
+import nl.tudelft.cse1110.andy.config.RunConfiguration;
+import nl.tudelft.cse1110.andy.execution.step.GetRunConfigurationStep;
+import nl.tudelft.cse1110.andy.result.Result;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class RunConfigurationTests extends IntegrationTestBase {
```

Comment:
These are unit tests, and they will look nicer once the rule is inside the `RunConfiguration` and you can simply call a `config.isValid()` rather than making a static method call to somewhere else